File: abc/abc296/b.py
# https://atcoder.jp/contests/abc296/submissions/40279568

a = list(range(8, 0, -1))
b = [chr(i) for i in range(97, 105)]
c = [input() for _ in range(8)]

for y in range(8):
  for x in range(8):
    if c[y][x] == '*':
      print(b[x], a[y], sep = '')

File: abc/abc296/b.cpp
// https://atcoder.jp/contests/abc296/submissions/40215408

#include <bits/stdc++.h>
using namespace std;

int main() {
  char c[8][8];
  for(int i = 0; i < 8; i++) for(int j = 0; j < 8; j++) cin >> c[i][j];

  for(int i = 0; i < 8; i++) {
    for(int j = 0; j < 8; j++) {
      if(c[i][j] == '*') {
        cout << (char)(j + 97);
        cout << 8 - i;
      }
    }
  }
}
